articles/virtual-machines/virtual-machine-scale-sets-maintenance-control-template.md

```diff
@@ -94,14 +94,15 @@ For more information, see [configurationAssignments](/azure/templates/microsoft.
 
 ```json
 { 
-  "type": "Microsoft.Maintenance/configurationAssignments", 
-  "apiVersion": "2021-09-01-preview", 
-  "name": "string", 
-  "location": "string", 
-  "properties": { 
-    "maintenanceConfigurationId": "string", 
-    "resourceId": "string" 
-  } 
+"type": "Microsoft.Maintenance/configurationAssignments",
+"apiVersion": "2021-09-01-preview",
+"name": "[variables('maintenanceConfigurationAssignmentName')]",
+"location": "string (e.g. westeurope)", 
+"scope": "Resource Id of the resource that is being assigned to the Maintenance Configuration (e.g. VMSS Id)"
+"properties": {
+            "maintenanceConfigurationId": "Resource Id of the Maintenance Configuration"
```

Comment:
Looks overall good, but please tab this back to be in line similarly to how it was before (the indent is too deep). Do this for like 103 and 104. Thank you!